multi-config/templates/configs.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
There is a [size limit](https://kubernetes.io/docs/concepts/configuration/configmap/#:~:text=A%20ConfigMap%20is%20not%20designed,separate%20database%20or%20file%20service.) here that users should be aware of, definitely can hit it with a couple large configs.

You could alternatively create a ConfigMap for each of the files provided in the `vector/` dir.

Reply:
This one's kinda tricky honestly.

Though the `ConfigMap` templates definitely support the Go templating that would be necessary for that, the `values.yaml` that adds them to Vector does not. [It's a major issue that Helm has](https://github.com/helm/helm/issues/2492), and one that I call out in the `README`.

I'll a note about this to the README, even more evidence that Observability Pipelines has much better tooling for handling this. 😄

Reply:
True - though you could manually add them into to vector's `values.yaml` or do some awful shelling and seding to generate it before running helm commands